README.md

```diff
@@ -1,7 +1,47 @@
-# 📒 A MVP data catalog for the Federal Office for Agriculture (FOAG)
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)
+[![Pages](https://img.shields.io/github/deployments/blw-ofag-ufag/data-catalog/github-pages?label=GitHub%20Pages)](https://blw-ofag-ufag.github.io/data-catalog/)
+[![GitHub last commit](https://img.shields.io/github/last-commit/blw-ofag-ufag/data-catalog.svg)](https://github.com/blw-ofag-ufag/data-catalog/commits)
+[![GitHub issues](https://img.shields.io/github/issues/blw-ofag-ufag/data-catalog.svg)](https://github.com/blw-ofag-ufag/data-catalog/issues)
 
-Welcome to this MVP data catalog repository! This project is a first version of a data catalog, presenting the Federal Office for Agricultures (FOAG) data in a structured yet user-friendly way.
 
-The goal of this data catalog is to have a simple solution covering only basic requirements; but using a data model that is interoperable with [I14Y](https://www.i14y.admin.ch/) and [opendata.swiss](https://opendata.swiss).
+DigiAgriFoodCH Data Catalog
+===========================
 
-You can access the data catalog [here](https://blw-ofag-ufag.github.io/data-catalog/index.html?lang=de&sort=issued-desc). Please note that while all the source code and the metadata about our datasets are publically available, many hyperlinks to resources or people point to internal services with restricted access.
+Welcome to the FOAG Data Catalog, a user-friendly yet robust repository that showcases our diverse data assets in one place. It brings together the best of modern data management: **interoperability**, **openness**, **collaboration**, and **ease of access**.
```

Comment:
I think it's time to remove direct mentions of FOAG and have it more neutral, or at least mention that this is a BLW-BLV collaboration.
It is true that most of the work is BLW, but nevertheless...